src/components/icon/Dashboard.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Icon from './Icon.js';
+
+export default function Dashboard({ active, ...props }) {
+  return (
+    <Icon viewBox="0 0 80 80" {...props}>
+      <g id="dashboard" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+        <g transform="translate(15.000000, 15.000000)">
+          <path
+            d="M44.1940916,42.8409798 C39.1471856,48.2809744 32.3243291,50.9988316 25.5026399,50.999999 C18.6358126,51.0019444 11.7701527,48.2490682 6.71546422,42.7464286 C-2.32071543,32.9100031 -2.4012636,17.3429259 7.46841436,7.47381612 C12.4526718,2.48984564 18.9774612,-0.00116689225 25.5026399,4.10061094e-07 C32.0247055,0.00116771237 38.5467711,2.49295843 43.5286938,7.47381612 C53.3217149,17.2662734 53.4295015,32.8882136 44.1940916,42.8409798 Z"
+            id="Path"
+            fill={active ? '#69D5B1' : '#BFBFBF'}
```

Comment:
I wonder if we could declare `const` variables for the hex codes? @crysfel what do you think?

Reply:
I think that would be nice to have, that way we can easily update the colors if we decide to update our theme or something. (A search/replace is not a big deal though)

Reply:
I think find-replace might be better for now. I can't think of any truly meaningful, reusable names for a variable here. They seem like one-offs to me.